### PR TITLE
:sparkles: Allow editing e-mail of the user in profile screen.

### LINKED ISSUE #671: Authentification + GitHub et "Keep my email addresses private"
Hello,

J'utilise la fonctionnalité "Keep my email addresses private" sur GitHub, et j'ai utilisé GitHub pour m'identifier comme speaker. Si on m'envoie un email, je ne peux pas le recevoir, de fait.

image

Est-il possible de personnaliser l'email (et/ou d'être invité·e à indiquer un email pour communiquer entre organisateurices et speaker) depuis l'interface de Conference Hall ?

### PR DESCRIPTION
**Pull request by GerardPaligot**
Closes https://github.com/bpetetot/conference-hall/issues/671

Small contribution but there's a beginning for everything. :)

### CODE REVIEW

File: src/screens/components/profile/profile.jsx [hunk 1]
```diff
@@ -35,6 +35,7 @@ const Profile = ({
             inline
           />
           <Field name="company" label="Company" type="text" component={input} inline />
+          <Field name="email" label="Email" type="text" component={input} inline />
```
Comment (bpetetot) [inline src/screens/components/profile/profile.jsx]: Can you put it between the field `fullname` and `company`?
Can you make it required (like the `fullname`)?

**Review by bpetetot — COMMENTED** [review]
Thanks a lot for the PR 🙏
I just have a small change

**Comment by GerardPaligot** [comment]
Fixed and rebased. Should be ok. Thanks @bpetetot for your review!
I'll try to make a bigger pull request soon :)

**Review by bpetetot — APPROVED** [review]
Thanks @GerardPaligot

File: src/screens/components/profile/profile.jsx (hunk 1 above)
Comment (thom4parisot) [reply src/screens/components/profile/profile.jsx]:
```suggestion
          <Field name="email" label="Email" type="email" component={input} inline />
```

Surtout pour des questions de [clavier de saisie sur mobile](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/email) (et aussi pour la validation native)

**Pull request merged**